include/deal.II/grid/tria.h

```diff
@@ -3483,6 +3483,34 @@ class Triangulation : public Subscriptor
     std::pair<std::pair<cell_iterator, unsigned int>, std::bitset<3>>> &
   get_periodic_face_map() const;
 
+  /**
+   * Translate the unique id of a coarse cell to its index.
+   *
+   * @note: For serial and shared triangulation both id and index are the same.
```

Comment:
doxygen will render this funny with the colon:
```suggestion
   * @note For serial and shared triangulation both id and index are the same.
```

Reply:
OK!